src/systemctl/systemctl.c

```diff
@@ -3987,6 +3987,38 @@ static void print_status_info(
         else
                 printf("\n");
 
+        if (endswith(i->id, ".timer")) {
+                dual_timestamp nw;
+                char *unit_path;
+                char tstamp1[FORMAT_TIMESTAMP_RELATIVE_MAX], tstamp2[FORMAT_TIMESTAMP_MAX];
+                char *next_rel_time = NULL, *next_time = NULL;
+
+                printf("  Trigger: ");
+
+                dual_timestamp_get(&nw);
+                unit_path = unit_dbus_path_from_name(i->id);
```

Comment:
you are leaking "unit_path" here. Use `_cleanup_free_` on the declaration to free it automatically when it goes out of scope